Upstatement/puppy-lib
# Update linting configs and bump node version to 14.16.0

## Changes

- Bump ESLint & Prettier config versions to 1.0.0 (and update peer dependencies)
- Add pre-commit hooks
- Update node version to 14.16.0 (the version of Puppeteer we're now using [requires 10.18.1+](https://www.npmjs.com/package/puppeteer#usage) and I was getting errors in the puppy-template repo so I figured I'd just bump to the most recent LTS version)

## Steps to test

See steps to test in [puppy template PR](https://github.com/Upstatement/puppy/pull/181)

## Notes

- The diff is only so big because of the `package-lock.json` updates 😱
